rumqttc/src/lib.rs

```diff
@@ -121,12 +121,12 @@ pub type Incoming = Packet;
 /// Current outgoing activity on the eventloop
 #[derive(Debug, Eq, PartialEq, Clone)]
 pub enum Outgoing {
-    /// Publish packet with packet identifier. 0 implies QoS 0
-    Publish(u16),
-    /// Subscribe packet with packet identifier
-    Subscribe(u16),
-    /// Unsubscribe packet with packet identifier
-    Unsubscribe(u16),
+    /// Publish packet with packet identifier and topics. 0 implies QoS 0
+    Publish(u16, Vec<String>),
```

Comment:
Maybe it would be more readable to give these fields names now that there is more than one? Like
```
Publish{
  packet_id: u16,
  topics: Vec<String>,
}
```

Reply:
Yep that makes sense. I've gone with `pkid` for parity with the named fields on the incoming structs.